tests/playwright-test/command-line-filter.spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test, expect } from './playwright-test-fixtures';
```

Comment:
Let's move `should focus a single test spec`, `should focus a single nested test spec` and `should focus a single test suite` here?

Reply:
This test is about command line, as opposed to "only" aka focus. So I'd keep them separate, those are orthogonal aspects that should not be confused.